daemon/api_test.go

```diff
@@ -68,8 +69,9 @@ func (s *apiSuite) All() ([]snappy.Part, error) {
 	return s.parts, s.err
 }
 
-func (s *apiSuite) Find(searchTerm string) ([]snappy.Part, error) {
+func (s *apiSuite) Find(searchTerm string, channel string) ([]snappy.Part, error) {
```

Comment:
(searchTerm, channel string)

That shorter syntax may be used for all for the changes below.